packages/frontend-plugin-api/src/wiring/createExtension.ts

```diff
@@ -122,6 +122,7 @@ export type CreateExtensionOptions<
   UFactoryOutput extends ExtensionDataValue<any, any>,
 > = {
   kind?: TKind;
+  /** @deprecated namespace is no longer required, you can safely remove this option and it will default to the `pluginId`. It will be removed in a future release. */
   namespace?: TNamespace;
```

Comment:
Did this end up working and showing up correctly in editors? Otherwise I'm thinking we should log a warning or something like that too